emission/core/wrapper/common_trip.py

```diff
@@ -0,0 +1,20 @@
+import logging
+import emission.core.wrapper.wrapperbase as ecwb
+
+class CommonTrip(ecwb.WrapperBase):
+    props = {"start_loc" : ecwb.WrapperBase.Access.WORM, # _id of common_place of starting point
+             "end_loc" : ecwb.WrapperBase.Access.WORM, # _id of common_place of ending point
+             "trips" : ecwb.WrapperBase.Access.WORM, # List of trip_ids that are associated with this common trip
+             "probabilites" : ecwb.WrapperBase.Access.WORM,  # a matrix that represents the probabilites for edge
```

Comment:
I'd really like to understand how this matrix is structured, and how I can retrieve the probability of an edge, etc.